components/drops/CreateDrop.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="drop-card border card-border-color">
```

Comment:
card sould be clickable? I would use `nuxt-link` then

Reply:
Since we linking outside the app, would it be best to use an `a` tag in this case ?

Reply:
Still possible to do [without](https://nuxt.com/docs/api/components/nuxt-link#target-and-rel-attributes)

Reply:
> Still possible to do [without](https://nuxt.com/docs/api/components/nuxt-link#target-and-rel-attributes)

I've experimented this with NuxtLink and its "to" attribute. However, I noticed that when using it, the link is appended to the current path, which might not always be the desired behavior we want. Not sure if it only happens for me.



https://github.com/kodadot/nft-gallery/assets/60175531/b7aee63b-794d-47ca-9930-529ab1fe9629

Reply:
Doesn't work with nuxt-bridge..